README.md

```diff
@@ -62,17 +70,44 @@ Username: postgres
-7) If you'd like to get the current project running, you can clone the repository with `git clone https://github.com/gannonprudhomme/AutoScheduler/` and run the steps below.
+
+8) Navigate the the frontend directory: `./autoscheduler/frontend/src/` and run `npm install`
+    - This is for TypeScript compilation and testing.
+    
+9) Before making code changes, its recommended to goto the github project [wiki](https://github.com/aggie-coding-club/Rev-Registration/wiki) and follow the Project Setup and Install section.
 
 ## Running
 
 Before running any commands, if you're not running in the virtual environment(you should see `(env)` somewhere in your current terminal line), run `source env/bin/activate` or `./env/Scripts/activate`.
 
-You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+**Server:**
+- You will have to make and apply migrations before running the server (and whenever our models are changed). To generate the files that django uses to apply migrations, run `./manage.py makemigrations`, and to then apply these to the database, run `./manage.py migrate`.
+    - `manage.py` can be found in the `./autoscheduler/` directory.
+
+- After migrating, you can run the server with `./manage.py runserver`. When running for the first time, you will have to configure `autoscheduler/autoscheduler/config/postgres-info.json` with your postgres username and password. This file will be automatically created if you run `./manage.py runserver`.
+
+- If `./manage.py` isn't working, try to use `python manage.py ...` or `python3 manage.py ...` or `py manage.py ...` in place of it.
+    - See python aliases above
+
+- The first time you run the server, you will need to scrape the course info from TAMU
+    - `python manage.py scrape_depts`
+    - `python manage.py scrape_courses`
+    - `python manage.py scrape_grades`
+
+**Frontend:**
+- Navigate to the frontend source directory: `cd ./autoscheduler/frontend/src` and run `npm run dev`
+    - This sets up a watch for file changes and compiles them
+    - ctrl-c to quit
+
+**`localhost:8000`**
+- Open in web browser after doing the above steps.
 
-After migrating, you can run the server with `./manage.py runserver`. When running for the first time, you will have to configure `autoscheduler/autoscheduler/config/postgres-info.json` with your postgres username and password. This file will be automatically created if you run `./manage.py runserver`.
+**Testing:**
 
-If `./manage.py` isn't working, try to use `python manage.py` or `python3 manage.py` in place of it.
+- The frontend has automated test suites which can be ran with:
+    - `cd ./autoscheduler/frontend/src`
+    - `npm run test`
+        - Can put filename after to test only one file.
```

Comment:
Does this actually work? I usually end up running `npx jest <filename>` because I could never get `npm run test <filename>` to work for me

Reply:
It works on my computer, do you normally run ``npx jest <filename>`` in the frontend/src directory? I'll add that